### DISCUSSION TITLE
I want to help

### DISCUSSION THREAD

**Discussion by motorto** [robske110/IDDataLogger#61] [General]
Hey,

Would love to help this project.

Would be super cool if you created maybe a roadmap of features/bug fixes needed.

Thanks

**Comment by robske110** [comment]
Hi, thank you for your interest (and sorry for not replying).
There are some things features that would make this more complete:
- The odometer value is being logged in the db, but it is not displayed anywhere yet.
- The API now gives location info, but it is currently not saved to the database (or requested)
- (Very large feature/task) This odometer value (and location info) could be used to automatically calculate trips
- Exports (for example for the charging history, etc.)
